app/assets/stylesheets/homepage.scss

```diff
@@ -0,0 +1,10 @@
+// Place all the styles related to the Homepage controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+body {
+    background-color: rosybrowns;
+    font-family: proxima-nova, 'Helvetica Neue', Arial, Helvetica, sans-serif;
+  }
```

Comment:
I am unsure where this body scss comes from. I may delete it and see if it changes anything.